components/cli/pkg/util/lang.go

```diff
@@ -83,6 +84,31 @@ func (langMgr *BLangManager) Init() error {
 }
 
 func (langMgr *BLangManager) GetExecutablePath() (string, error) {
+	balVersionCmdOutput := ""
+	cmd := &exec.Cmd{}
+	cmd = exec.Command("ballerina", "version")
```

Comment:
Can we make this inline?
cmd := exec.Command("ballerina", "version")

Reply:
Updated.